bayesian_cnn_prometheus/learning/model/experiment_setup.py

```diff
@@ -1,4 +1,6 @@
 from sacred import Experiment
 
+from bayesian_cnn_prometheus.constants import Paths
+
 ex = Experiment()
-ex.add_config("/home/alikrz/Pulpit/MyStuff/cancer-detection/3d-cnn-prometheus/bayesian_cnn_prometheus/learning/config.json")
+ex.add_config(str(Paths.CONFIG_PATH.resolve()))
```

Comment:
Z ciekawości, to czy napewno jest potrzebny ten `str`?

Reply:
Tak, niestety metoda "_create_config_dict", która jest w "add_config" nie wspierała mi zmiennych Path i nie konwertowała ich samemu na stringa. Obsługują kilka typów, a w innych choćby dzałały wyrzucają wyjątek:
TypeError: Invalid argument type <class 'pathlib.PosixPath'>

Takżę pathlib nie jest wspierana